src/components/modules/ui.ts

```diff
@@ -750,6 +750,8 @@ export default class UI extends Module<UINodes> {
       return;
     }
 
+    this.Editor.BlockManager.setCurrentBlockByChildNode(focusedElement);
```

Comment:
This method will be called many times when user will do selecting some text fragment. Maybe we can provide some optimizations?